packages/paypal-commerce-utils/src/paypal-commerce-types.ts

```diff
@@ -96,9 +103,34 @@ export type PayPalSdkComponents = Array<
  *
  */
 export interface PayPalFastlaneSdk {
+    ThreeDomainSecureClient: {
+        isEligible(params: threeDSecureParameters): Promise<boolean>;
+        show(): Promise<ThreeDomainSecureClientShowResponse>;
+    };
     Fastlane(options?: PayPalFastlaneOptions): Promise<PayPalFastlane>;
 }
 
+interface ThreeDomainSecureClientShowResponse {
+    liabilityShift: string;
+    authenticationState: string;
```

Comment:
Would be better to set enum type up here with available states instead of `string`. The same action for `liabilityShift` and use these enums where needed

Reply:
Fixed

Reply:
For authenticationState you need to use the same approach